data/json/monsters/mutant.json

```diff
@@ -110,6 +110,7 @@
     "morale": 13,
     "regen_morale": true,
     "speed": 150,
+    "attack_cost": 130,
```

Comment:
They're designed around being able to move in and attack on the same turn consistently.
```suggestion
```